src/traffic_server/InkIOCoreAPI.cc

```diff
@@ -157,6 +157,7 @@ TSThreadCreate(TSThreadFunc func, void *data)
   if (!tid) {
     return (TSThread) nullptr;
   }
+  thread->tid = tid;
```

Comment:
This isn't set by `ink_thread_trampoline`?

Reply:
I don't think it is, the `tid` was always set to 0 when I checked.

Reply:
Trying to be subtle here, but could you look at whether that's a better place to set `tid` and make a PR that fixes that if so?